docs/content/rules/sort-interfaces.mdx

```diff
@@ -410,12 +410,52 @@ Current API:
 
 You can define your own groups and use regexp patterns to match specific interface members.
 
-Each key of `customGroups` represents a group name which you can then use in the `groups` option. The value for each key can either be of type:
-- `string` — An interface member's name matching the value will be marked as part of the group referenced by the key.
-- `string[]` — An interface member's name matching any of the values of the array will be marked as part of the group referenced by the key.
-The order of values in the array does not matter.
+A custom group definition may follow one of the two following interfaces:
 
-Custom group matching takes precedence over predefined group matching.
+```ts
+interface CustomGroupDefinition {
+  groupName: string
+  type?: 'alphabetical' | 'natural' | 'line-length' | 'unsorted'
+  order?: 'asc' | 'desc'
+  selector?: string
+  modifiers?: string[]
+  elementNamePattern?: string
+}
+```
+An interface member will match a `CustomGroupDefinition` group if it matches all the filters of the custom group's definition.
```

Comment:
[minor] missing line.

Reply:
Thanks! Fixed in a79be0392458b9e25f35b30826a848e65f6b40f0.